prac_07/guitar.py

```diff
@@ -0,0 +1,17 @@
+"""CP1404/CP5632 Practical - Guitar class example."""
+
+
+class Guitar:
+    def __init__(self, name="", year=0, cost=0):
```

Comment:
`cost=0.0`
Making the default a float helps PyCharm and the programmer know it should be a float.